ecl/provider_test.go

```diff
@@ -21,7 +21,6 @@ var (
 	OS_CONTAINER_INFRA_ENVIRONMENT           = os.Getenv("OS_CONTAINER_INFRA_ENVIRONMENT")
 	OS_DB_DATASTORE_TYPE                     = os.Getenv("OS_DB_DATASTORE_TYPE")
 	OS_DB_DATASTORE_VERSION                  = os.Getenv("OS_DB_DATASTORE_VERSION")
```

Comment:
Probably these are not necessary too:)

Reply:
Since there is no part which is using, it deletes. Thank you.